src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -121,7 +125,7 @@
      [list/section-list
       {:sections   transactions-history-sections
        :key-fn     :hash
-       :render-fn  render-transaction
+       :render-fn  (partial render-transaction keycard-account?)
```

Comment:
I think Andrey added `render-data` option to avoid closure creation on lists as it was a render performance botleneck